apps/console/src/features/identity-providers/components/wizards/oidc-authentication-provider-create-wizard.tsx

```diff
@@ -128,13 +125,16 @@ export const OidcAuthenticationProviderCreateWizard: FunctionComponent<MinimalAu
                     const location = response.headers.location;
                     const createdIdpID = location.substring(location.lastIndexOf("/") + 1);
 
-                    onIDPCreate(createdIdpID);
+                    history.push({
```

Comment:
This will redirect users to the `edit` page. This approach is okay for IDP view but not for sign-in methods section. 
Let's use `onIDPCreate(createdIdpID);`.